docs/changelog.rst

```diff
@@ -4,7 +4,7 @@
 Changelog
 =========
 
-Next Version
```

Comment:
I think it's good to keep the `next version` section, pointing out that we should always update the changelog.

Reply:
I am following the process in the `realease.md` file.
The 'Next Version' section will be re-introduced in the second to last step.